app/src/main/java/net/skyscanner/backpack/demo/stories/DialogStory.kt

```diff
@@ -54,101 +54,75 @@ class DialogStory : Story() {
 
   private val dialogsByType = mapOf(
     "Normal" to { context: Context ->
-      val dialog = BpkDialog(context)
+      val dialog = BpkDialog(context, BpkDialog.Type.Success)
       dialog.apply {
         title = "You are going to Tokyo!"
         description = "Your flight is all booked. Why not check out some hotels now?"
-        icon = BpkDialog.Icon(
-          R.drawable.bpk_tick,
-          context.getColor(R.color.bpkMonteverde),
-        )
+        icon = BpkDialog.Icon(R.drawable.bpk_tick)
 
         addActionButton(
-          BpkButton(context).apply {
-            text = "Continue"
-            setOnClickListener {
-              println("confirmed")
-              dialog.dismiss()
-            }
+          BpkDialog.Button("Continue") {
+            println("confirmed")
```

Comment:
should we remove these? they look like some debug code that got accidentally commited